### DISCUSSION TITLE
Icon Pack

### DISCUSSION THREAD

**Discussion by fr0zon** [Blackymas/NSPanel_HA_Blueprint#548] [Q&A]
Hi all, 
Looking for the icon codes and pics. 
Was linked a YT Video but the link is broken.

**Comment by fr0zon** [comment]
Not to worry i found it - 
https://htmlpreview.github.io/?https://github.com/jobr99/Generate-HASP-Fonts/blob/master/cheatsheet.html

**Reply by deejaybeam** [reply]
If you use version 3.2.1 - this link is also in the blueprint itself
